### ISSUE TITLE
Resolving SmartEnum values from multiple assemblies

### ISSUE THREAD

**Issue by DennisInSky** [ardalis/SmartEnum#93]
It would be nice to configure a list of assemblies which should be used for resolving derived smart enum types sitting in different assemblies.

**Comment by ardalis** [comment]
You mean something like how `.AddMediatR` takes in a list of assemblies for resolving Handlers? I could see that. Would need to configure it in a platform-neutral fashion, but then we could ship DI-specific helpers for AspNetCore, etc...

**Comment by DennisInSky** [comment]
Right, something like this as it may turn out one wants to have enum values sitting in derived classes which are part of the other assemblies. I thought exactly what you suggest, i.e. having something like services.AddSmartEnums(listOfAssemblies)
